src/css/components/actions.scss

```diff
@@ -6,12 +6,11 @@
 
 .action-link {
   color: $color-primary;
+  font-size: $sans-s2;
   font-weight: bold;
   text-decoration: underline;
 
-  &:visited {
-    color: $color-primary;
-  }
+  &:visited { color: $color-primary; }
```

Comment:
Please keep the formatting to what it previously was as that's whats compliant with the CSS styleguide.